main.html

```diff
@@ -1,138 +1,171 @@
-
-                        </table>
-                    </div>
-                </div>
-
-                <!-- create or edit or delete task -->
-                <div class="new-task-box">
-                    <h2>Add New Task</h2>
-
-                    <form action="/action_page.php">
-                        <label for="title">Title:</label><br>
-
-                        <input class="input" type="text" id="title"
-                            name="title"><br>
-
-                        <label for="time">Category:</label><br>
-
-                        <div class="category-row-box">
-                            <button type="button" class="category-health">
-                                <img class="category-logo"
-                                    src="./img/family-icon.png"
-                                    alt="todo
-                                    logo" />
-                            </button>
-
-                            <button type="button" class="category-family">
-                                <img class="category-logo"
-                                    src="./img/heart-icon.png"
-                                    alt="todo
-                                    logo" />
-                            </button>
-
-                            <button type="button" class="category-job">
-                                <img class="category-logo"
-                                    src="./img/job-icon.png"
-                                    alt="todo
-                                    logo" />
-                            </button>
-
-                            <button type="button" class="category-study">
-                                <img class="category-logo"
-                                    src="./img/study-icon.png"
-                                    alt="todo
-                                    logo" />
-                            </button>
-
-                            <button type="button"
-                                class="category-entertainment">
-                                <img class="category-logo"
-                                    src="./img/entertainment-icon.png"
-                                    alt="todo
-                                    logo" />
-                            </button>
-
-                        </div>
-
-                        <label for="time">Time:</label><br>
-
-                        <input class="input" type="time" id="time" name="time"><br>
-                        <button class="submit-button" name="submit-button"
-                            type="button" value="HTML"><span>Save</span></button>
-                        <button class="delete-button" name="delete-button"
-                            type="button" value="CSS"><span>Delete</span></button>
-                    </form>
-                </div>
-            </div>
+            <label for="time">Time:</label>
+
+            <input class="input" type="time" id="time" name="time" />
```

Comment:
- Class `input` does not explain that much. What if you wanna introduce new input style? Then you have to refactor it all. Don't follow bootstrap's naming convention. That makes sense when you want to build a ui library only.
- Are you using id anywhere?